### PR TITLE
Make kubectl available from everywhere

### PR DESCRIPTION
**Pull request by bronhaim**
Set your kubevirt_path by the following and you'll be able to use
kubectl from everywhere:

export KUBEVIRT_PATH=".../Projects/kubevirt/kubevirt/"
alias kubectl="$KUBEVIRT_PATH/cluster/kubectl.sh"

I assume you run all clusters from same kubevirt directory in this
patch.

Signed-off-by: Yaniv Bronhaim <ybronhei@redhat.com>

### CODE REVIEW

File: cluster/kubectl.sh [hunk 1]
```diff
@@ -1,19 +1,21 @@
 #!/bin/bash
 
-source hack/config.sh
+source ${KUBEVIRT_PATH}hack/config.sh
 
-SYNC_CONFIG=cluster/vagrant/sync_config.sh
+SYNC_CONFIG=${KUBEVIRT_PATH}cluster/vagrant/sync_config.sh
```
Comment (rmohr) [inline cluster/kubectl.sh]: You also need KUBEVIRT_PATH in sync_config.sh to make it consistent.

**Comment by rmohr** [comment]
@bronhaim have been thinking about this one:

What if you would add something like 

```bash
if [ -n "$KUBEVIRT_PATH" ]; then
    cd $KUBEVIRT_PATH
fi
```

on top of `cluster/kubectl.sh` and `cluster/sync.sh` instead of adding the path always everywhere?

**Comment by bronhaim** [comment]
@rmohr :\ The idea was to run kubectl from every path once you set the kubevirt_path. Jumping to the directly is not the desire.. Anyhow, I don't find it much useful anymore, as we also have the vagrant env files there so most of the time I switch to the directory anyway and use separate shell for kubectl commands. if you don't find it useful too much you can close the pull request

**Comment by rmohr** [comment]
@bronhaim I find it nice and useful, but I think we should limit it to cluster/kubectl.sh and cluster/vagrant/config_sync.sh.

**Pull request merged**